src/t2iapi/device/service.proto

```diff
@@ -121,4 +121,11 @@ service DeviceService {
     not able to maintain the static state, it shall return RESULT_NOT_SUPPORTED.
      */
     rpc SetBatteryUsage (t2iapi.device.SetBatteryUsageRequest) returns (BasicResponse);
+	
+	 /*
+    Provide information about the pm:CalibrationInfo for the given pm:AbstractDeviceComponentDescriptor.
```

Comment:
I would expect the wording for last calibration, otherwise it can not distinguish between next calibration